L1Trigger/TrackFindingTracklet/data/memorymodules_hourglassExtendedAllCombined.dat

```diff
@@ -0,0 +1,2721 @@
+AllInnerStubs:        AS_D1PHIA_DM            [42]
```

Comment:
This file is also changed by https://github.com/cms-L1TK/cmssw/pull/301 . Why do we have two PRs changing the same file?